docs/source/hub.rst

```diff
@@ -8,12 +8,14 @@ Publishing models
 Pytorch Hub supports publishing pre-trained models(model definitions and pre-trained weights)
 to a github repository by adding a simple ``hubconf.py`` file;
 
-``hubconf.py`` can have multiple entrypoints. Each entrypoint is defined as a python function with
-the following signature.
+``hubconf.py`` can have multiple entrypoints. Each entrypoint is a pre-trained model you want to
+publish, defined as a python function with the following signature.
 
 ::
 
     def entrypoint_name(pretrained=False, *args, **kwargs):
+        # pretrained is required keyword argument.
```

Comment:
I didn't realized we always had pretrained as a required keyword argument. Do you remember? Shall we relax?

Reply:
Sure we can relax it! It was my impression that we prefer to have pretrained weights to make it more ready to use for users, but yea you are right this shouldn't be required.